### DISCUSSION TITLE
Directories not found

### DISCUSSION THREAD

**Discussion by spencerpaul132** [cyberbotics/webots#5339] [Q&A]
I am attempting to use webots in my classroom with 28 students. One of the problems I am running into is a few students when they attempt to create a new world, when they click finish it will say "some directories could not be found" and not allow them to create a new project.

Please advice.

**Comment by stefaniapedrazzi** [comment]
Which version of Webots are they using? On which platform?
Are they using the `New World File..` or the `New Project Directory..` wizard?
What is the path where they are trying to create "a new world"?
Could you report exactly what is the error message printed in the console?

World files needs to be located in valid Webots project directories.
Also make sure to have the writing rights in the chosen target directory otherwise the new files cannot be created.

**Comment by spencerpaul132** [comment]
We are on windows, and we select project directory and I notice that there are no "documents" in the writing when it creates a new file. It just says C:\users\my_project

and then they name the world and click finish and then it says, "Some Directories could not be found" and it takes them back to their previous world.

Thank you,


Spencer Paul

Computer Science

Oldham County Middle School

502-222-1451

[cid:f8eb2733-f955-457c-8e88-6e1afeed1022][cid:269a4be8-3268-4281-94f7-e717b9ee24b6]


________________________________

**Comment by stefaniapedrazzi** [comment]
When using the `New Project Directory..` wizard you should not use the suggested path but select a valid path.
`C:\Users` is not a good path where to store data (and it is not accessible without administrator rights).
Instead you should choose for example `C:\Users\<CurrentUserName>\my_project`.

Meanwhile we will inspect why Webots suggested the invalid `C:\Users` as project path.

**Comment by stefaniapedrazzi** [comment]
Is it possible that the username of the students that have issues contain some special characters?
Are you using the latest version of Webots R2022b?
